src/controllers/scripting/legacy/controllerscriptinterfacelegacy.cpp

```diff
@@ -158,24 +158,25 @@ void ControllerScriptInterfaceLegacy::setValue(
                 coScript->getKey(), ControlFlag::AllowMissingOrInvalid);
         if (pControl &&
                 !m_st.ignore(
-                        pControl, coScript->getParameterForValue(newValue))) {
+                        pControl, coScript->getNormalizedValueForValue(newValue))) {
```

Comment:
Soft takeover deals with the position of the slider, not with the business logic value in Mixxx. That's we we need to calculate here the position on the slider form the value.